.github/workflows/perf-test.yml

```diff
@@ -0,0 +1,158 @@
+name: Performance Test
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - "**"
+
+jobs:
+  performance_test:
+    name: Run performance tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ipfs/aegir/actions/cache-node-modules@master
+        with:
+          directories: ./packages/perf/node_modules
+
+      - name: Install playwright dependencies
+        uses: ipfs/aegir/actions/cache-node-modules@master
+      - run: npx playwright install-deps
```

Comment:
I don't think this step is necessary any more, playwright-test seems to install it's own deps now.